stylesheet.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
+body {
+  background-color: #181818;
+  width: 100%;
+  height: 100vh;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  color:white;
+  display:grid;
+  place-items: center;
+}
+
+.login page {
+  width: 400px;
+  align-items: center;
+}
+
+
+
+form{
+  width: 256px;
+  display: flex;
+  box-shadow: 0px 5px 15px 0px rgba(0, 0, 0, 0.43);
+  flex-direction: column;
+  border: 1px solid white;
+  margin-bottom: 16px;
+
+}
+
```

Comment:
Add some margin bottom to the inputs in the form.

Reply:
Can you please explain this in detail.

Reply:
Currently the form inputs/labels are too close together, so adding `margin-bottom` to your inputs would solve this.